src/ventus/wm/window.js

```diff
@@ -486,8 +489,23 @@ function(Emitter, Promise, View, WindowTemplate) {
 		},
 
 		move: function(x, y) {
-			this.x = x;
-			this.y = y;
+			var maxX = window.innerWidth - this._margin.right;
+			var maxY = window.innerHeight - this._margin.bottom;
```

Comment:
This has to be addressed to, we can't assume Ventus is taking the whole screen size in every situation